x-pack/plugin/security/cli/src/main/java/org/elasticsearch/xpack/security/cli/CertGenUtils.java

```diff
@@ -186,6 +188,38 @@ public static X509Certificate generateSignedCertificate(
         );
     }
 
+    /** GenerateSignedCertificate() Redifinition to restrict Exntended Key Usage by adding the boolean forServerAuth  **/
+    public static X509Certificate generateSignedCertificate(
```

Comment:
We don't want to just copy-and-paste another version of this method.
Since the only change is to add another parameter we can either delegate the old method to the new one, or replace all the existing callers to pass `false` as the parameter.

Duplicated code like this leads to diverging implementations which can become a source of bugs.